### PR TITLE
[FORMATS-146] Add variant quality field.

### LINKED ISSUE #146: Add quality field to variant
Needed for https://github.com/bigdatagenomics/avocado/issues/253

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #146.

### CODE REVIEW

File: src/main/resources/avro/bdg.avdl [hunk 1]
```diff
@@ -765,6 +765,12 @@ record Variant {
   union { null, string } alternateAllele = null;
 
   /**
+   The Phred scaled probability that this allele is variant. VCF column 6.
```
Comment (heuermh) [inline src/main/resources/avro/bdg.avdl]: minor nit, `VCF column 6.` &rarr; `VCF column 6 "QUAL".`

**Comment by heuermh** [comment]
Then about releases, this will be required for ADAM 0.23.0, so we cut a 0.11.2 for bdg-formats with only this PR in it?

**Comment by fnothaft** [comment]
Addressed review comments.

> Then about releases, this will be required for ADAM 0.23.0, so we cut a 0.11.2 for bdg-formats with only this PR in it?

Yup!

**Comment by heuermh** [comment]
Thank you, @fnothaft!

**Pull request merged**